pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[project]
+name = "dsst_etl"
+dynamic = ["version"]
+description = "An ETL package for data processing at DSST"
+requires-python = ">=3.11"
+
+
+dependencies = [
+    "sqlalchemy",
```

Comment:
I have tried to keep all pre-existing dependencies listed in the repository and adding our new ones. We want all python dependencies to be tracked in this file.